mindmap/static/js/src/mindmap.js

```diff
@@ -19,4 +21,17 @@ function MindMapXBlock(runtime, element) {
 
     var jm = new jsMind(options);
     jm.show(mind);
+
+    $(element).find('.save-button').click(function () {
+        const mindMapData = jm.get_data('node_array');
+        const jsonMindMapData = JSON.stringify(mindMapData);
+        var handlerUrl = runtime.handlerUrl(element, 'upload_file');
```

Comment:
const handlerUrl = runtime.handlerUrl(element, 'upload_file');

Please change variables that will be used  one time by `const`